### PR TITLE
[showcase script] Adds new URLs to the blocklist

### PR DESCRIPTION
**Pull request by sarah11918**
This PR adds some new sites to the showcase blocklist as they were not built with Astro.

### CODE REVIEW

File: scripts/update-showcase.mjs [hunk 1]
```diff
@@ -435,6 +435,17 @@ const scraper = new ShowcaseScraper({
 		"https://souto.tk",
 		"https://unwrapped.studio/",
 		"https://gdscyu.com/",
+		"https://ahuja-lab.in/",
+		"https://roudstudio.com/",
+		"https://oengi.com/",
+		"https://kireerik.github.io/refo/",
+		"https://aiaman.web.app/",
+		"https://zenorocha.com/",
+		"https://www.rmmanfredi.com/",
```
Comment (TheOtterlord) [inline scripts/update-showcase.mjs:444]: This looks to have Astro hints
image
```suggestion
```

File: scripts/update-showcase.mjs [hunk 2]
```diff
@@ -435,6 +435,17 @@ const scraper = new ShowcaseScraper({
 		"https://souto.tk",
 		"https://unwrapped.studio/",
 		"https://gdscyu.com/",
+		"https://ahuja-lab.in/",
+		"https://roudstudio.com/",
+		"https://oengi.com/",
+		"https://kireerik.github.io/refo/",
+		"https://aiaman.web.app/",
```
Comment (TheOtterlord) [inline scripts/update-showcase.mjs:442]: Also astro. It just has a weird customisation that isn't picked up.
image
```suggestion
```

**Review by TheOtterlord — COMMENTED** [review]
Found two Astro sites, but the rest lgtm

**Comment by sarah11918** [comment]
@TheOtterlord Thanks! I guess I have two more sites to add 😅

**Pull request merged**